### ISSUE TITLE
Translations from Crowdin are so wrong

### ISSUE THREAD

**Issue by Maks220v** [maruohon/tweakeroo#529]
I don't know why, but somebody definitely trolled the whole translation for all Malilib-related mods. I'll include some obvious trolling, but the overall translation level is very-very poor...

![изображение](https://github.com/user-attachments/assets/37dba146-cd97-4fbd-bc53-73f75de7db00)

**Comment by Maks220v** [comment]
For context: "ЖильеОфферферферфрферферферферферферферента Книги" means "Villag-offerferferferferferferentaBooks"

Also "Показать предложения жителей села для зачарованных книг" means something like "Show trades from villagers of a village for enchanted books". That's some obvious copy-pasta from google translator, or some stupid AI copy-pasta :D

**Comment by sakura-ryoko** [comment]
AI + Google Translate was used.  Your welcome to send me fixed versions of the JSON files.  If you need help, ask me.

**Comment by Maks220v** [comment]
> AI + Google Translate was used. Your welcome to send me fixed versions of the JSON files. If you need help, ask me.

Yeah! I'll be very proud to translate the whole mod, or all mods to my language, if i'll know how :D

I used to translate one github app, but developer just sent me a list of placeholders, so i could write my own translation on top. Could you tell me where to look so i can translate all the text? Is it in JSON files of a mod?

**Comment by sakura-ryoko** [comment]
It's all in JSON key files, take a look here for MiniHUD (1.21.4 branch) -->
https://github.com/sakura-ryoko/minihud/tree/15529dc80330ba14bb7db7464b0a4b96cbf2b5c5/src/main/resources/assets/minihud/lang

If you wish, I can also send you the "broken" Crowdin file.

**Comment by Maks220v** [comment]
> It's all in JSON key files, take a look here for MiniHUD (1.21.4 branch) --> https://github.com/sakura-ryoko/minihud/tree/15529dc80330ba14bb7db7464b0a4b96cbf2b5c5/src/main/resources/assets/minihud/lang
> 
> If you wish, I can also send you the "broken" Crowdin file.

Oh wowiee! I will do my best to translate, thank you for locating files for me <3
I'll sent you translated files asap!

**Comment by Maks220v** [comment]
I just translated Litematica to start with something, and my next goal is Item Scroller. I'm seeing a lot of text on the descriptions of functions, so it will take some time to translate as well. Sharing my progress until i'll head for evening!
![изображение](https://github.com/user-attachments/assets/60e637ac-0e92-44e3-b3da-8bbc1f34dbc0)
[en_US.json](https://github.com/user-attachments/files/17991885/en_US.json)

**Comment by sakura-ryoko** [comment]
In the future, can you post the files in the related Repo Issues pages, under my Repos?  I do monitor these; but just asking.

**Comment by sakura-ryoko** [comment]
Also the file that you've attached is for Syncmatica.

**Comment by Maks220v** [comment]
> In the future, can you post the files in the related Repo Issues pages, under my Repos? I do monitor these; but just asking.

Yep! Meant syncmatica, was late for me, srry ^^

Will post where you asked, no problem

**Comment by sakura-ryoko** [comment]
There also is an existing Russian lang file for Syncmatica that I was recently sent, so I think that makes you the third person to have sent me Russian for Syncmatica, tbh.

**Comment by sakura-ryoko** [comment]
Please check the current build of it ... what version are you using?  I don't always build JAR's for Syncmatica right away, but they've been added to my repo.

**Comment by Maks220v** [comment]
> Please check the current build of it ... what version are you using? I don't always build JAR's for Syncmatica right away, but they've been added to my repo.

I've been busy for the whole last week, so i'm sorry for being inactive! I used Syncmatica v. 0.3.12-sakura.6 because i mainly use 1.21.1 for my server with friends :^)

Will try to translate other mods a bit faster, so expect new translated files soon 😄 
Also, i'm quite new on Github so sorry for posting things in wrong places in advance, xddd

**Comment by sakura-ryoko** [comment]
Okay so here are the Repo Links that can use Translation Assistance:

MaLiLib (No Russian) --> https://github.com/sakura-ryoko/malilib/tree/pre-rewrite/fabric/1.21.4/src/main/resources/assets/malilib/lang
Litematica (No Russian) --> https://github.com/sakura-ryoko/litematica/tree/pre-rewrite/fabric/1.21.4/src/main/resources/assets/litematica/lang
MiniHUD (No Russian) --> https://github.com/sakura-ryoko/minihud/tree/pre-rewrite/fabric/1.21.4/src/main/resources/assets/minihud/lang
Tweakeroo (No Russian) --> https://github.com/sakura-ryoko/tweakeroo/tree/pre-rewrite/fabric/1.21.4/src/main/resources/assets/tweakeroo/lang
ItemScroller (No Russian) --> https://github.com/sakura-ryoko/itemscroller/tree/pre-rewrite/fabric/1.21.4/src/main/resources/assets/itemscroller/lang
Syncmatica (Recently updated Russian) --> https://github.com/sakura-ryoko/syncmatica/tree/1.21.4/src/main/resources/assets/syncmatica/lang
Litematica-Printer (No Russian) --> https://github.com/sakura-ryoko/litematica-printer/tree/1.21.4/src/main/resources/assets/litematica-printer/lang

They don't have Russian files now because of similar complaints, so I now require a Human translator for even a single line; or else it will contain English.  So, with that being said; someone would need to download the `en_us.json` file, perform the translation; and send it back to me.  I don't care if someone uses a PR, Posts it in a `Issues` post, or sends it to me on Discord... as long as I get sent the corrected JSON file.
